src/main/resources/default/assets/tycho/styles/forms.scss

```diff
@@ -0,0 +1,35 @@
+/*!
+ * Made with all the love in the world
+ * by scireum in Remshalden, Germany
+ *
+ * Copyright by scireum GmbH
+ * http://www.scireum.de - info@scireum.de
+ */
+
+// --------------------------
+// Token Autocomplete
+// --------------------------
+
+.form-control.token-autocomplete-container {
+  align-content: center;
+}
+
+.token-autocomplete-container .token-autocomplete-token {
+  border-radius: 4px;
```

Comment:
Could we reduce the border-radius to 0 oder maybe 2 to match the general style of Tycho?